_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+comments: true
+title:  "CARLA 0.9.13 Release"
+subtitle: "User defined traffic manager paths, carla-agents Python package, user defined pedestrian animations and new vehicle behaviours"
+description: "User defined traffic manager paths, carla-agents Python package, user defined pedestrian animations and new vehicle behaviours"
+author: "@MattRowe18"
+image: 'img/carla.jpg'
+background: '/img/posts/2021-08-02/town10.png'
+---
+
+{% include youtube.html id="tWAg1fIDNCg" %}
+
+{% include release_button.html release_tag="0.9.13" %}
+
+The __0.9.13 release of CARLA__ is finally here, so read on to see how the new improvements will revolutionize your workflow. 
```

Comment:
As a general note, I feel like there phrase (with its variants) "It is useful for training and evaluating for autonomous driving" is repeated way too much in the blog